src/js/helpers.js

```diff
@@ -1,3 +1,8 @@
-function getView (view){
-    return 'views/'+view+'/'+view+'.html';
-}
\ No newline at end of file
+/**
+ * Retrieve the string URL of a view or directive template
+ * @param  {string} filename Name of the html file to retrieve
+ * @return {string}          Relative URL to the minified template
+ */
+function getView (filename){
+  return `dist/html/${filename}.min.html`;
```

Comment:
This is the ES2015 test. Use template strings to build the url instead of the old-fashioned way.

Also added a header. I think for helper functions like this we should do an expanded header, since they are more like an API than the angular codebase.